src/app/equipment-car/equipment-car.component.html

```diff
@@ -5,15 +5,17 @@
             <mat-card-header class="title">
                 <mat-card-title class="title-info">Information about car</mat-card-title>
             </mat-card-header>
-            <img mat-card-image src="" alt="photo of car">
+
             <mat-card-content>
-                <p>Car name: {{carData.carName}} </p>
-                <p>Type: {{carData.type}}</p>
-                <p>Marking:{{carData.marking}}</p>
-                <p>Destiny: {{carData.destiny}}</p>
-                <p>Operational number:{{carData.operationalNumber}}</p>
-                <p>Person responsible for the car:{{carData.employee.surname}} {{carData.employee.name}}</p>
+                <p>Car name: {{carData?.carName}} </p>
+                <p>Type: {{carData?.type}}</p>
+                <p>Marking: {{carData?.marking}}</p>
+                <p>Destiny: {{carData?.destiny}}</p>
+                <p>Operational number: {{carData?.operationalNumber}}</p>
+                <p>Person responsible for the car: {{carData?.employee?.surname}} {{carData?.employee?.name}}</p>
+                <img class="image-car" mat-card-image [src]="carData?.link" alt="photo of car">
```

Comment:
atrybut `alt` powinien mieć jakiś znaczacy opis. W tym momencie każde auto będzie miało "photo of the car". Proponuje aby tutaj dodać nazwę samochodu, który jest wyświetlany.
Atrybut alt jest wazny z powodu accessibility. Zachęcam do poczytania na ten temat https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices